src/AppInstallerCLICore/Workflows/PortableFlow.cpp

```diff
@@ -187,16 +191,36 @@ namespace AppInstaller::CLI::Workflow
                 }
                 else
                 {
-                    entries.emplace_back(std::move(PortableFileEntry::CreateFileEntry(entryPath, targetPath, {})));
+                    if (isSymlinkCreationSupported)
+                    {
+                        entries.emplace_back(std::move(PortableFileEntry::CreateFileEntry(entryPath, targetPath, {})));
+                    }
+                    else
+                    {
+                        // Find matching nested installer file and rename the symlink target with the corresponding command alias if available.
+                        for (const auto& nestedInstallerFile : nestedInstallerFiles)
+                        {
+                            if (relativePath == std::filesystem::path{ ConvertToUTF16(nestedInstallerFile.RelativeFilePath) })
```

Comment:
== just does raw string comparison (does not consider case insensitive, ..\, .\, etc), make sure it fits what we need here